src/controllers/breaches.js

```diff
@@ -189,13 +176,13 @@ async function bundleVerifiedEmails (options) {
     : []
 
   for (const breach of foundBreachesWithRecency) {
-    // add resolved status from v1: breach_resolved
+    // add resolved status from v1: breaches_resolved
     breach.IsResolved = !!resolvedBreachesV1.includes(breach.recencyIndex)
```

Comment:
Can/should we use lowercase "i", i.e. breach.isResolved?

Reply:
I think we are keeping it consistent with the rest of the booleans from HIBP